charts/app-config/values-integration.yaml

```
@@ -980,6 +980,7 @@ govukApplications:
 
 - name: feedback
   helmValues:
+    cspReportOnly: false
     extraEnv:
```

GitHub still doesn't support suggestions on lines that aren't already modified, so I'll use `feedback` to demonstrate the change you want, even though `feedback` isn't one of the apps whose environment you want to change.

You're gonna need to specify this on every app that cares about the flag (except the three where you're removing the flag, naturally).

I know it's a bit ugly, but it's the best we can do given that this flag is inverted in its sense (i.e. it means "disable this new feature" rather than "enable this new feature"). Alternatively you could flip the sense of the flag in its implementation so that you can just enable the feature app-by-app instead of having to disable it in most apps and then gradually un-disable it 🙂

```suggestion
    extraEnv:
      - name: GOVUK_CSP_REPORT_ONLY
        value: "true"
```